parsers/jscript.c

```diff
@@ -1755,47 +1756,58 @@ static void parseFunction (tokenInfo *const token)
 		is_generator = true;
 		readToken (name);
 	}
+
 	if (isType (name, TOKEN_OPEN_PAREN))
 	{
 		/* anonymous function */
 		copyToken (token, name, false);
-		anonGenerate (name->string, "anonymousFunction", JSTAG_FUNCTION);
+		anonGenerate (name->string, "anonymousFunction_", JSTAG_FUNCTION);
```

Comment:
Do we need this in this commit?

If you think we need this one, let's make a commit for introducing "_".
You can prepend such a commit to this pull request.
In such a case, I would like you to write the background or reason why "_" is needed in its commit log. I don't say introducing "_" breaks compatibilities. However, I don't say introducing "_" is a small change. Unlike "refactor", it is user-visible change. Therefore, I would like you to explain this to users.